lib/grape/middleware/formatter.rb

```diff
@@ -82,6 +82,7 @@ def read_rack_input(body)
                 env['rack.request.form_input'] = env['rack.input']
               end
             rescue StandardError => e
+              raise e if e.is_a? Grape::Exceptions::Base
```

Comment:
Shouldn't it be:

    rescue Grape::Exception::Base => e
        raise e
    rescue StandardArray => e
        ....

If we don't raise the error originating from the parser it can never be rescued and the follwing calls are likely to experience other errors?